pkg/config/features.go

```diff
@@ -15,4 +15,5 @@ const (
 	FeatureMetricsInstance  = "metrics-instance"
 	FeatureFileWatcher      = "file-watcher"
 	FeatureAgentAPI         = "agent-api"
+	FeatureDataplaneHealth  = "dataplane-health"
```

Comment:
Don't think you need to add this feature. The constant isn't being used anywhere.

Reply:
Thought, I'd removed it, added it initially.